Hw24/Vector2d.cpp

```diff
@@ -0,0 +1,147 @@
+#include "Vector2d.h"
+
+Vector2d::Vector2d(const float x, const float y):
+m_x1{x},
+m_y1{y}
+{
+}
+
+Vector2d::Vector2d(const float x1, const float y1, const float x2, const float y2):
+m_x1{ x1 },
+m_y1{ y1 },
+m_x2{ x2 },
+m_y2{ y2 }
```

Comment:
should setup two values from these 4 values

Reply:
changed